toolchest_client/api/query.py

```diff
@@ -111,6 +112,9 @@ def run_query(self, tool_name, tool_version, input_prefix_mapping,
         :param skip_decompression: Whether to skip decompression of the output file, if it is an archive.
         :param instance_type: instance type that the tool will run on.
         :param volume_size: size of the volume for the instance.
+        :param universal_volume_name: the name of the volume that should be attached to the current run or persisted
+            after.
+        :param universal_name: the name of the universal function to pull parameters from.
```

Comment:
Nit: "name of universal volume" and "name of universal function" would suffice here